packages/cdktf/lib/util.ts

```diff
@@ -96,3 +97,32 @@ export function keysToSnakeCase(object: any): any {
     return newObject;
   }, {});
 }
+
+/**
+ * dynamic attributes are located at a different position than normal block attributes
+ * This method detects them and moves them from .attributeName to .dynamic.attributeName
+ * It also invokes the .toTerraform() method on the dynamic attribute to get the correct
+ * Terraform representation
+ *
+ * TODO: this does not move nested blocks (do they exist at all?)
```

Comment:
Can we detect them and throw?

Reply:
no need for that, as this was an obsolete todo (we now support nested blocks)